README.md

```diff
@@ -72,6 +72,26 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] While you did a good job deploying and customizing your README file :+1:, as required by the previous reviewer kindly make sure to remove the comments like `Describe between 1-3 key features of the application.` from your readme :+1: